tests/acceptance/test_files/mod_install.bats

```diff
@@ -74,6 +74,16 @@ local
   assert_output --partial 'does not satisfy mod.m4 which requires version 10.99.99'
 }
 
+@test "install a mod with protocal in url" {
+  steampipe mod install https://github.com/turbot/steampipe-mod-hackernews-insights@0.3.0
```

Comment:
You need to prefix this statement with a `run` for `bats` to be able to pick up the output.

Otherwise, `assert_output` does not work.

Reply:
@Mr-Destructive ^^^